packages/jest-repl/src/cli/repl.ts

```diff
@@ -5,30 +5,34 @@
  * This source code is licensed under the MIT license found in the
  * LICENSE file in the root directory of this source tree.
  *
- * @flow
  */
 
-import type {GlobalConfig, ProjectConfig} from 'types/Config';
-
-declare var jestGlobalConfig: GlobalConfig;
-declare var jestProjectConfig: ProjectConfig;
-declare var jest: Object;
+declare var jestGlobalConfig: Config.GlobalConfig;
+declare var jestProjectConfig: Config.ProjectConfig;
 
 import path from 'path';
 import repl from 'repl';
 import vm from 'vm';
+import {Transformer} from '@jest/transform';
+import {Config} from '@jest/types';
 
-let transformer;
+let transformer: Transformer;
 
-const evalCommand = (cmd, context, filename, callback, config) => {
+const evalCommand = (
+  cmd: string,
+  _context: any,
+  _filename: string,
+  callback: (e: Error | null, result?: any) => void,
+  _config: Config.GlobalConfig,
```

Comment:
Not sure what type this is since we already have `jestProjectConfig` and `jestGlobalConfig`